.travis.yml

```diff
@@ -4,3 +4,5 @@ node_js:
 # https://github.com/apiaryio/protagonist/issues/81
 #  - "stable"
   - "0.12"
+before_install: npm install -g grunt-cli
+after_script: grunt test-browser-sauce
```

Comment:
you don't need to install grunt-cli
just put command into package json and run `npm run test-browser-sauce`